client/src/types.ts

```diff
@@ -1,20 +1,13 @@
-export type UserRecord = {
```

Comment:
tady pozor, mas trochu konflikt v souborech. Mas soubor types.ts a generovany types.d.ts a to dela problem - protoze TS si mysli, ze jsou "jeden a ten samy" (zjednodusene). Doporucuju tenhle prejmenovat napr. na eventTypes.ts a pak zmenit obsah na
```ts
import type { components } from "./types";


export type Event = components["schemas"]["Event"];
export type DateRecord = components["schemas"]["DateRecord"];
export type Vote = components["schemas"]["Vote"];

export type EventsResponse = { items: Event[] };

export type NewEventPayload = {
    title: string;
    location?: string;
    dates: number[];
};
```